changelog/unreleased/hide-file-versions-shares.md

```diff
@@ -0,0 +1,5 @@
+Change: Hide file versions for share receivers
+
+We needed to change the visibility of file versions and hide them to share receivers. Space Editors can still see and restore file versions.
```

Comment:
Could you elaborate how this is intended to evolve in the future? The change is substaintial as it breaks the CI but solves a very specific problem. Maybe this should be configurable?

Reply:
This is decided to be the new default product behavior.